openfoodfacts/utils.py

```diff
@@ -0,0 +1,13 @@
+import requests
+
+API_URL = "http://world.openfoodfacts.org/"
+
```

Comment:
In a lib it's not recommende to use settings file. What I propose instead is to allow to change this url via a function.